pipeline/outputs/s3.md

```diff
@@ -649,3 +650,57 @@ The following example uses `pyarrow` to analyze the uploaded data:
 3  2021-04-27T09:33:56.539430Z    0.0     0.0       0.0         0.0          0.0            0.0
 4  2021-04-27T09:33:57.539803Z    0.0     0.0       0.0         0.0          0.0            0.0
 ```
+
+## Enable Parquet support
+
+### Build requirements for Parquet
+
+To enable Parquet, build Fluent Bit with Apache Arrow support and install Arrow GLib/Parquet GLib:
+
+```bash
+# Ubuntu/Debian example
+sudo apt-get update
+sudo apt-get install -y -V ca-certificates lsb-release wget
+wget https://packages.apache.org/artifactory/arrow/$(lsb_release --id --short | tr 'A-Z' 'a-z')/apache-arrow-apt-source-latest-$(lsb_release --codename --short).deb
+sudo apt-get install -y -V ./apache-arrow-apt-source-latest-$(lsb_release --codename --short).deb
+sudo apt-get update
+sudo apt-get install -y -V libarrow-glib-dev libparquet-glib-dev
+
+# Build Fluent Bit with Arrow:
+cd build/
+cmake -DFLB_ARROW=On ..
+cmake --build .
+```
+
+For other Linux distributions, refer [the document for installation instructions of Apache Parquet](https://arrow.apache.org/install/).
+Apache Parquet GLib is a part of Apache Arrow project.
+
+### Testing Parquet compression
+
+```md
```

Comment:
```suggestion

```

looks like that was accidental? Committing this should trigger the vale tests to run again and that should ( I think) clear the suggestions.